controllers/front/CartController.php

```diff
@@ -105,15 +105,8 @@ public function initContent()
         if (!Tools::getValue('ajax')) {
             $this->checkCartProductsMinimalQuantities();
         }
-        $presenter = new CartPresenter();
-        $presented_cart = $presenter->present($this->context->cart, $shouldSeparateGifts = true);
```

Comment:
By default it's:

`public function present($cart, $shouldSeparateGifts = false)`

And in the `FrontController`:
`'cart' => $this->cart_presenter->present($this->context->cart),`

Reply:
@kpodemski Keep it off, gift separation is not working and results in wrong totals - https://github.com/PrestaShop/PrestaShop/issues/20564.

That's why the gifts are turned off in FrontController.